Library/Homebrew/.rubocop.yml

```diff
@@ -34,6 +34,7 @@ Naming/PredicateMethod:
 Naming/MethodParameterName:
   AllowedNames:
     - go
+    - flatpak
```

Comment:
Are you sure this is needed?

Reply:
@castrojo Good context, thanks!
@ahmedadan What's the failure without this line?